lib/network/api/resource/operation.go

```diff
@@ -21,20 +20,16 @@ func NewOperation(bo *block.BlockOperation) *Operation {
 }
 
 func (o Operation) GetMap() hal.Entry {
-	body, _ := operation.UnmarshalBodyJSON(o.bo.Type, o.bo.Body)
-
 	return hal.Entry{
-		"hash":    o.bo.Hash,
-		"source":  o.bo.Source,
-		"type":    o.bo.Type,
-		"tx_hash": o.bo.TxHash,
-		"body":    body,
+		"hash":   o.bo.Hash,
+		"source": o.bo.Source,
+		"type":   o.bo.Type,
```

Comment:
I think this code should not change from sync branch. 

 https://github.com/bosnet/sebak/blob/master/lib/network/api/resource/operation.go#L23